main_flow/main_flow.py

```diff
@@ -53,24 +52,13 @@ def loadCache():
     model_instruction = keras.models.load_model(file_instru)
     model_ingredients = keras.models.load_model(file_ingri)
 
-    return model_instruction and model_ingredients and top_instru_dict and top_ingri_dict
+    return model_instruction and model_ingredients and top_instruc_dict and top_ingred_dict
 
 
 
 
 TOP_WORD_NUM = 1500
 EXTRA_FEATURES_NUM = 14
```

Comment:
I assume u didn't move it as well because main_flow uses it? 
in this case, main_flow can take it from your new file :) [we want to avoid same constant spread acros the globe like DBZ...